### PR TITLE
ENG-1082: Column lineage title improvements

### PR DESCRIPTION
**Pull request by havenbarnes**
image

### CODE REVIEW

**Comment by ianrtracey** [comment]
Awesome! 🙌

File: frontend/src/components/lineage/LineageView.tsx [hunk 1]
```diff
@@ -397,29 +398,34 @@ export function LineageView({ lineage, rootAsset }: { lineage: Lineage, rootAsse
...
+            fallback={
+              <div className="bg-muted flex flex-col h-full text-muted-foreground font-semibold  justify-center  items-center">
+                <div className="max-w-[550px]">
+                  <ErrorFallback />
+                </div>
+              </div>
+            }
+          >
+            <ReactFlowProvider>
+              <TooltipProvider>
```
Comment (havenbarnes) [inline frontend/src/components/lineage/LineageView.tsx:423]: @ianrtracey better place for the tooltip provider to go? I searched for a bit but didn't find anything obvious. Figure we probably want one app-wide though!

Comment (ianrtracey) [reply frontend/src/components/lineage/LineageView.tsx:423]: That's a good question. How about `app/layout.tsx` ?

**Review by ianrtracey — APPROVED** [review]

File: frontend/src/components/lineage/LineageView.tsx (hunk 1 above)
Comment (havenbarnes) [reply frontend/src/components/lineage/LineageView.tsx]: Sounds good, updated!

**Pull request merged**